### ISSUE TITLE
Staging Review finding: "Curly quote" apostrophe can't be read by screen reader in plain text version of email notifications

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#94352]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Veteran Facing Forms
**Product:** Forms Status
**Feature:** Form Status on My VA & Confirmation Page

## Findings details
**VA.gov Experience Standard - issue:** [User experiences an unexpected interaction with a component or pattern.](https://design.va.gov/about/experience-standards/#user-experiences-predictable-interactions-with-components)
**VA.gov Experience Standard - category:** [Usability](https://design.va.gov/about/experience-standards/#category-usability)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

Since HTML emails with table-based layouts can be challenging for screen reader users, some screen reader users configure their mail clients to read all messages in plain text only (for example, [plain text mode in Outlook](https://support.microsoft.com/en-us/office/read-email-messages-in-plain-text-16dfe54a-fadc-4261-b2ce-19ad072ed7e3)). This sidesteps any bad HTML that may be creating a barrier to reaching the content of the message.

Listening with a screen reader, the plain text versions of the messages are pretty comprehensible --- except when there's an apostrophe. The "curly quote" style apostrophes are announced as their character code, eg. "We can't process" gets announced as "We can=E2=80=99t process."

The actual impact of this is probably low. Anecdotally, fewer screen reader users are setting up their mail clients this way in recent years, and I think it's safe to assume that anyone who has configured their mail client this way is probably used to hearing unexpected artifacts like this. But it seems like a point of friction we should be able to smooth out and could avoid confusion.

<details><summary>Link, screenshot or steps to recreate</summary>

![H1 heading in an email reading "Action needed: We can't process your lay or witness statement to support a VA claim" with a curly-quote apostrophe used in "can't."](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/staging-review-images/80272-Brian-4.png?raw=true)

</details>

## Recommended action

Two possible solutions to explore with the VA Notify team:
- When populating the `text/plain` version of the message, strip out any curly quotes or other cosmetic special characters and replace them with their plain text equivalents.
- Just use plain text characters across the board in email notifications, both the HTML and plain text versions. Note that this conflicts with the [content style guide's punctuation guidance](https://design.va.gov/content-style-guide/punctuation#quotation-marks), so it may need a bigger conversation with folks in CAIA and elsewhere in VA.

## References

- **Accessibility Defect Severity:** [4: Low. Consider fixing or exploring in 2-4 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-4)
- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by tbaker1026** [comment]
Jeana to validate to see if it is still happening
